docs/source/install.rst

```diff
@@ -31,24 +38,69 @@ Another way to achieve this is to pass the ``--recursive`` option to the ``git
 clone`` command which will automatically initialize and update DTKData in the
 DataTransferKit repository.
 
+Building DTK
+------------
+
+DTK is configured and built using TriBITS <https://tribits.org>. DTK builds
+within Trilinos effectively as an extension package. First, link DTK into the Trilinos main directory:
+
+.. code::
+
+    $ cd $TRILINOS_DIR
+    $ ln -s $DTK_DIR
+
+DTK can then be configured along with the rest of Trilinos using
+TriBITS. TriBITS is an extension of CMake which allows DTK to be configured
+with scripts such as:
+
+.. code-block:: bash
+                
+  cmake \
+    -D CMAKE_INSTALL_PREFIX:PATH=$INSTALL_DIR \
+    -D CMAKE_BUILD_TYPE:STRING=DEBUG \
+    -D CMAKE_VERBOSE_MAKEFILE:BOOL=OFF \
+    -D BUILD_SHARED_LIBS:BOOL=OFF \
+    -D TPL_ENABLE_MPI:BOOL=ON \
+    -D TPL_ENABLE_Boost:BOOL=ON \
+    -D Boost_INCLUDE_DIRS:PATH=$BOOST_INCLUDE_DIR \
+    -D TPL_ENABLE_Libmesh:BOOL=OFF \
+    -D TPL_ENABLE_Netcdf:BOOL=OFF \
+    -D TPL_ENABLE_MOAB:BOOL=OFF \
+    -D TPL_ENABLE_BinUtils:BOOL=OFF \
+    -D Trilinos_ENABLE_ALL_OPTIONAL_PACKAGES:BOOL=OFF \
+    -D Trilinos_EXTRA_REPOSITORIES="DataTransferKit" \
+    -D Trilinos_ASSERT_MISSING_PACKAGES=OFF \
+    -D Trilinos_ENABLE_EXPLICIT_INSTANTIATION:BOOL=ON \
+    -D Trilinos_ENABLE_CXX11:BOOL=ON \
+    -D Trilinos_ENABLE_DataTransferKit:BOOL=ON \
+    -D Trilinos_ENABLE_DataTransferKitSTKMeshAdapters:BOOL=OFF \
+    -D Trilinos_ENABLE_DataTransferKitMoabAdapters:BOOL=OFF \
+    -D Trilinos_ENABLE_DataTransferKitLibmeshAdapters:BOOL=OFF \
+    -D Trilinos_ENABLE_DataTransferKitClassicDTKAdapters:BOOL=ON \
+    -D DataTransferKit_ENABLE_DBC:BOOL=ON \
+    -D DataTransferKit_ENABLE_TESTS:BOOL=ON \
+    -D DataTransferKit_ENABLE_EXAMPLES:BOOL=ON \
+    -D DataTransferKit_ENABLE_ReadTheDocs:BOOL=ON \
+    -D DataTransferKit_ENABLE_ClangFormat:BOOL=ON \
```

Comment:
I would not advertise `DataTransferKit_ENABLE_ReadTheDocs` and `DataTransferKit_ENABLE_ClangFormat`